Networking/Sources/Networking/TargetType.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  File.swift
+//  
+//
+//  Created by marwa on 18/06/2022.
+//
+
+import Foundation
+import Alamofire
+
+public enum HTTPMethodd: String {  //to map alamofire http methods so if there any change in alamofire I will change here only
```

Comment:
```suggestion
/// Used to map alamofire http methods.
///
/// - Note: so if there any change in alamofire, we need to update it here. 
public enum HTTPMethodd: String { 
```
This is how to document a type.